### PR TITLE
Some new Safe loot

### PR DESCRIPTION
**Pull request by DeityLink**
safeloot

:cl:
* rscadd: Added a couple new items to the Safe loot table.
* bugfix: Fixed emitter goggles not working on mannequins
* bugfix: Fixed glasses mapped onto mannequins not being properly equipped when the map loads

### CODE REVIEW

**Comment by help-maint** [comment]
The nanofloxacin pills are a little lame, especially given that the safe already has nanobots as a possible spawn. It's not even enough nanofloxacin to electrolyze into something else.

File: code/modules/maps/spawners/spawners.dm [hunk 1]
```diff
@@ -1198,7 +1199,10 @@
 	/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
 	/obj/item/weapon/dnainjector/nofail/randompower,
 	/obj/item/weapon/gun/syringe/rapidsyringe,
-	/obj/item/voucher/free_item/medical_safe
+	/obj/item/voucher/free_item/medical_safe,
+	/obj/item/weapon/storage/pill_bottle/nanofloxacin,
+	/obj/item/weapon/gun/portalgun,
```
Comment (help-maint) [inline code/modules/maps/spawners/spawners.dm]: Sure that you meant to put these in the medical spawner?

**Comment by Eneocho** [comment]
Does this replace the utterly disappointing vox safe's loot? See #28092.

**Comment by help-maint** [comment]
No

File: code/modules/maps/spawners/spawners.dm (hunk 1 above)
Comment (DeityLink) [reply code/modules/maps/spawners/spawners.dm]: woops

**Comment by DeityLink** [comment]
> 
> 
> The nanofloxacin pills are a little lame, especially given that the safe already has nanobots as a possible spawn. It's not even enough nanofloxacin to electrolyze into something else.

fair enough

> 
> 
> Does this replace the utterly disappointing vox safe's loot? See #28092.

sounds like a mapping issue

File: code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -652,11 +663,18 @@ var/list/science_goggles_wearers = list()
 	update_emitter()
 
 /obj/item/clothing/glasses/emitter/proc/update_emitter()
-	if (!emitter || !isturf(emitter.loc) || emitter.lying)
+	if (!emitter || !isturf(emitter.loc))
 		if (beam)
 			qdel(beam)
 			beam = null
 		return
+	if (ismob(emitter) && emitter:lying)
```
Comment (help-maint) [inline code/modules/clothing/glasses/glasses.dm]: Looks like you accidentally left the : operator here, which is redundant as it gets checked below anyways

Comment (DeityLink) [reply code/modules/clothing/glasses/glasses.dm]: yep

**Comment by GreatBigFailure** [comment]
Now I want the portal gun to be accessible outside of bus. I want it real bad.
It should not be accessible outside of bus. I need to say my peace, this is a bad idea. Please do it anyway.

**Comment by DeityLink** [comment]
> 
> 
> Now I want the portal gun to be accessible outside of bus. I want it real bad.
> It should not be accessible outside of bus. I need to say my peace, this is a bad idea. Please do it anyway.

There's no reason for it to not be accessible outside of bus beyond me being too lazy to do it. I originally planned it to be a higher tier of nanotrasen R&D tech. But for now this will do.

**Comment by Eneocho** [comment]
> 
> There's no reason for it to not be accessible outside of bus beyond me being too lazy to do it. I originally planned it to be a higher tier of nanotrasen R&D tech. But for now this will do.

You'd link it to a higher NT Experimental level? Currently, the maximum level you can lawfully reach is NT 3, if you deconstruct 2 Scientist's Serendipities with 50u on the glass, but that's a nonstandard source of research and it bugs the RnD console until you remove the glass or kick the deconstructor. How would an NT Ex level 2, 3, 4 or 5 be normally acquired?

**Comment by Kurfursten** [comment]
This seems better served using the lazy event system instead of two new atom-level movement procs, but consult with @DamianX to see if it suits needs

**Pull request merged**